src/dataProvider.js

```diff
@@ -0,0 +1,41 @@
+import { fetchUtils } from "react-admin";
+import { api } from "./conf";
+
+const apiUrl = `${api.defaults.baseURL}/admin`;
+const httpClient = fetchUtils.fetchJson;
+
+export default {
+  getList: (resource, params) => {
+    console.log(api.defaults.baseURL);
```

Comment:
```suggestion
```

Reply:
Oh, I'd like you to show how you did this!